src/main/java/net/coderbot/iris/shaderpack/ShaderPack.java

```diff
@@ -89,6 +98,35 @@ private static String readFile(Path path) throws IOException {
 		}
 	}
 
+	private Map<String, Map<String, String>> parseLangEntries(Path root) throws IOException {
+		Path langFolderPath = root.resolve("lang");
+		Map<String, Map<String, String>> allLanguagesMap = new HashMap<>();
+
+		if (!Files.exists(langFolderPath)) {
+			return allLanguagesMap;
+		}
+			Files.walk(langFolderPath, 1).filter(path -> !Files.isDirectory(path)).forEach(path -> {
+
+				Map<String, String> currentLanguageMap = Maps.newHashMap();
```

Comment:
One less import and actually fewer characters overall
```suggestion
				Map<String, String> currentLanguageMap = new HashMap<>();
```